src/components/invite-dialog/index.tsx

```diff
@@ -75,15 +70,6 @@ export class InviteDialog extends React.Component<Properties, State> {
           <Button onPress={this.writeInviteToClipboard} isDisabled={!this.props.inviteCode}>
             {this.state.copyText}
           </Button>
-
-          <div {...cn('remaining-invite-container')}>
```

Comment:
I don't think we want to remove this, do we? We still want to show this it just comes from a different number?

Reply:
Yeah mentioned above - `NOTE:: I notice the designs still contain the remaining count of invites displayed on the invite dialog. Are we sure we no longer require an invite count?`

If we're not going to return an `inviteCount` in the response (on the frontend referenced as `maxUses`) when fetching the invite, where is the number going to come from I'm not sure I understand this part. Unless the idea here was to replace `maxUses` with something like `inviteCount` or `remainingCount` or something..

Reply:
I'm confused, why wouldn't we return an inviteCount from the api? We have the count now, right? We just don't need invitesUsed and maxInvitesPerUser

Reply:
lol yeah sorry, some misunderstanding from yesterday 😆 thought it wasn't quite right removing the count in UI. 

Yeah so I'll re-add that part of the invite dialog and get the count from the `inviteCount` prop instead 🙂

Reply:
Made the changes and added inviteCount. There is a follow up PR that removes the `invitesUsed` prop.